main.cpp

```diff
@@ -19,13 +19,23 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
+    CastCallBack(Rewriter& rewriter):_rewriter(rewriter) {
     };
 
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+	auto cstmtptr = Result.Nodes.getNodeAs<clang::CStyleCastExpr>("cast");

```

Comment:
What if `cstmtptr` is a null pointer? How to protect the program from crashing in such cases?

Reply:
if(cstmtptr == nullptr) {return;}